examples/awsbraket.ipynb

```diff
@@ -128,13 +137,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Obtain and print the probabilies of the states\n",
     "prob_dict = eng.backend.get_probabilities(qureg)\n",
     "print(\"Probabilites for each of the results: \", prob_dict)"
    ]
   },
   {
    "source": [
-    "We can retrieve the result later on (of this job or a previously executed one) using the Task Arn provided when it was run. You have to remember the ammount of qubits involved in the job and the order you used."
+    "## Retrieve results form a previous execution\n",
```

Comment:
Small question: what actually happens if we pass a task ARN for a job that was never executed? Do we get an error?

Reply:
Once the job is successfully sent to Braket, the status and result of the job is updated in the corresponding Bucket folder. The possible status are

`status': 'CREATED'|'QUEUED'|'RUNNING'|'COMPLETED'|'FAILED'|'CANCELLING'|'CANCELLED',`

If we ask for a valid task Arn we will obtain the corresponding status. If the status is COMPLETED, we get the results. If the status is FAILED we get the failure reason as well (that is contained in the dictionary of the response to theAPI call) 

If we ask for a non valid Arn we get an error
`ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the GetQuantumTask operation: Quantum task dcdace15-40a4-4ee2-9956-555955565558 not found`

Reply:
Ok, so in case the Arn is not valid, the error message is somewhat understandable from AWS already. However, could you add a `try - except` block in the `retrieve` function inside `_awsbraket_boto3_client.py` to print out a nicer error message like "Unable to locate job with id XXXX" like what you did for the `send` function?